collections/ansible_collections/purestorage/pure1/plugins/modules/pure1_info.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2021, Simon Dodsley (simon@purestorage.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: pure1_info
+version_added: '1.0.0'
+short_description: Collect information from Pure1
+description:
+  - Collect information from a Pure1.
+  - By default, the module will collect basic
+    information including counts for appliances, volumes, filesystems, snapshots
+    and buckets. Fleet capacity and data reduction rates are also provided.
+  - Additional information can be collected based on the configured set of arguements.
+author:
+  - Pure Storage ansible Team (@sdodsley) <pure-ansible-team@purestorage.com>
+options:
+  gather_subset:
+    description:
+      - When supplied, this argument will define the information to be collected.
+        Possible values for this include all, minimum, appliances, tags
```

Comment:
`tags` is not accepted. Also tags/kvp's are not returned by the `info` module.

Reply:
True - I ended up not putting it in because the Pure1 APIs are not very good at expressing the tags from a high level. 
The logical thing would be to list available tags and show which arrays they are assigned to. I'll have another think about how to do this.